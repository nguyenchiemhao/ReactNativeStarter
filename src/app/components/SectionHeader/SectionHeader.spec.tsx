import React from 'react';
import SectionHeader from './SectionHeader';
import renderer from 'react-test-renderer';

describe('SectionHeader', () => {
  it('should render correctly', () => {
    const header = renderer.create(<SectionHeader title="Test" />).toJSON;
    expect(header).toMatchSnapshot();
  });
});
