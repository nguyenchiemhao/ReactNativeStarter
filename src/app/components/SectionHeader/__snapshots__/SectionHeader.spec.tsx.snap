// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader should render correctly 1`] = `[Function]`;
